### PR TITLE
Use latest java 8 and automatically upgrade in future

### PR DESCRIPTION
**Pull request by chbatey**
The "~" means it will get the latest version that is greater than
the one specified

### CODE REVIEW

**Review by 2m — APPROVED** [review]
LGTM. TIL about `~` in jabba.

File: .travis.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ scala:
 before_install:
   # using jabba for custom jdk management
   - curl -sL https://raw.githubusercontent.com/shyiko/jabba/0.11.1/install.sh | bash && . ~/.jabba/jabba.sh
```
Comment (2m) [inline .travis.yml]: Latest is `0.11.2` currently.

Comment (chbatey) [reply .travis.yml]: :+1:

**Review by raboof — APPROVED** [review]
I would like it even better if we could use a fixed version, but somehow automatically PR updating that version as new ones become available. Until that time this is great :+1:

**Pull request merged**